src/index.html

```diff
@@ -1,12 +1,298 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__content content">
+    <header class="header">  
+      <a class="img-link" href="#home">
+        <img 
+          src="./images/logo.png" 
+          alt="Logo" 
+          class="logo"
+        >
+      </a>
+      
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link is-active">Apple</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Samsung</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Smartphones</a>
+          </li>
+          <li class="nav-list__item">
+            <a 
+              data-qa="nav-hover" 
+              href="#" 
+              class="nav-list__link"
+            >
+            Laptops & Computers
+            </a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Gadgets</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Tablets</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Photo</a>
+          </li>
+          <li class="nav-list__item">
+            <a href="#" class="nav-list__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="content__cards-wrapper">
```

Comment:
it isn't bem methodology, block = content, element = cards, modificatory = wrapper it isn't correct